util/src/main/java/tc/oc/pgm/util/text/TextTranslations.java

```diff
@@ -310,9 +309,8 @@ public static String translate(String key, @Nullable CommandSender sender, Objec
     return LegacyComponentSerializer.legacySection().serialize(translate(text, locale));
   }
 
-  public static BaseComponent[] toBaseComponentArray(
-      Component component, @Nullable CommandSender viewer) {
+  public static String toMinecraftGson(Component component, @Nullable CommandSender viewer) {
     Component translated = translate(component, viewer == null ? SOURCE_LOCALE : getLocale(viewer));
-    return BungeeComponentSerializer.get().serialize(translated);
+    return GsonComponentSerializer.gson().serialize(translated);
```

Comment:
Need to use the color downsampling version?

Reply:
Oh, yeah you're right